accurics-aws-tf-sample.tf

```diff
@@ -231,23 +245,104 @@ resource "aws_dynamodb_table" "accurics-test-dynamodbtable1" {
     type = "S"
   }
   server_side_encryption {
-    enabled     = false
+    enabled = false
   }
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure rotation for customer created CMKs is enabled**
 
**Vulnerability**
Ensure rotation for customer created CMKs is enabled
 
**Remediation**
(From Console) In the navigation pane, choose Customer managed keys. Choose the alias or key ID of a CMK. Under General configuration, choose the Key rotation tab. (AWS Cli) Execute 'aws kms enable-key-rotation --key-id <KMS_KEY_ID>'

source: accurics-app